sssd_test_framework/utils/tools.py

```diff
@@ -476,6 +478,11 @@ def __init__(self, host: MultihostHost, fs: LinuxFileSystem) -> None:
         Run ``getent`` command.
         """
 
+        self.ssh: SSHKeysUtils = SSHKeysUtils(host, fs)
```

Comment:
Could naming it `ssh` be confusing with the authentication ssh() method?   Since it's for key utils, maybe naming it something like `sshKey` would be more descriptive and obvious?